### ISSUE TITLE
Add language switcher for EN/DE

### ISSUE THREAD

**Issue by Sivanajani** [Sivanajani/portfolio#2]
To increase accessibility and usability for a multilingual audience, a language switcher should be implemented using a library such as i18next.
